lib/active_merchant/billing/gateways/cyber_source_rest.rb

```diff
@@ -251,6 +251,63 @@ def add_merchant_description(post, options)
         merchant[:locality] = options[:merchant_descriptor_locality] if options[:merchant_descriptor_locality]
       end
 
+      def add_stored_credentials(post, payment, options)
+        return unless stored_credential = options[:stored_credential]
+
+        options = stored_credential_options(stored_credential, options.fetch(:reason_code, ''))
```

Comment:
Minor suggestion for readaiblity (`options` reads like the top-level hash, so it's a little confusing to redefine it)
```suggestion
        stored_credential = stored_credential_options(stored_credential, options.fetch(:reason_code, 
```